### PR TITLE
makefile cleanup

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
- build plugin for each platform and arch instead of a single one for all platforms
- removed unneeded tar packaging
- use yarn only instead of mix yarn and npm
- updated release github action pipeline

### CODE REVIEW

**Review by dmitryk-dk — COMMENTED** [review]
Overall looks good! Added somment

File: Makefile [hunk 1]
```diff
@@ -9,92 +9,82 @@ ifeq ($(PKG_TAG),)
...
+vm-frontend-plugin-release: \
+	vm-frontend-plugin-build \
+	vm-frontend-plugin-pack
 
-victoriametrics-datasource-plugin-release: \
-	victoriametrics-frontend-plugin-build \
-	victoriametrics-backend-plugin-build \
-	victoriametrics-datasource-plugin-pack
+vm-plugin-release: \
+	vm-plugin-pack-tar \
```
Comment (dmitryk-dk) [inline Makefile:79]: those two steps make a build of the package. As for me, this operation should only pack the plugin

Comment (AndrewChubatiuk) [reply Makefile:79]: it can lead to unpredictable consequences as broken bundle in result

**Comment by AndrewChubatiuk** [comment]
> As I can see, the build for the windows is incorrect

updated it as well, please check

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Pull request merged**